pkgs/os-specific/linux/nftables/python.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, setuptools
+, nftables
+}:
+
+buildPythonPackage {
+  pname = "nftables";
+  inherit (nftables) version src;
+
+  postPatch = ''
+    substituteInPlace "py/src/nftables.py" \
+      --replace-fail "libnftables.so.1" "${nftables}/lib/libnftables.so.1"
+  '';
+
+  setSourceRoot = "sourceRoot=$(echo */py)";
+
+  nativeBuildInputs = [ setuptools ];
+
+  pyproject = true;
+  pythonImportsCheck = [ "nftables" ];
+
+  meta = {
```

Comment:
We should probably just inherit the entire meta here

Reply:
`nftables.meta` contains `mainProgram`, which is not a thing for this package.